app/models/recommendation.rb

```diff
@@ -13,10 +13,20 @@ class ReviewRecommendationError < StandardError; end
   scope :reviewed, -> { where.not(reviewer_id: nil) }
   scope :submitted, -> { where(submitted: true) }
 
-  validate :reviewer_comment_is_present?
+  validate :reviewer_comment_is_present?, if: :review_complete?
 
   delegate :audits, to: :planning_application
 
+  enum(
+    status: {
+      assessment_in_progress: 0,
+      assessment_complete: 1,
+      review_in_progress: 2,
+      review_complete: 3
+    },
+    _default: :assessment_in_progress
+  )
```

Comment:
How about moving the default in the migration?

Reply:
Yes, good thought!

I wonder if there's any benefit to having both 🤔

Reply:
I think is better usually to have the default value on the creation on the database level and here you are extending the list of possible status 😃

Reply:
Yes, I think you're right. Amended!